localstack-core/localstack/services/ssm/provider.py

```diff
@@ -352,7 +352,12 @@ def _has_secrets(names: ParameterNameList) -> Boolean:
     @staticmethod
     def _normalize_name(param_name: ParameterName, validate=False) -> ParameterName:
         if is_arn(param_name):
-            return extract_resource_from_arn(param_name).split("/")[-1]
+            resource_name = extract_resource_from_arn(param_name).replace("parameter/", "")
+            # if the parameter name is only the root path we want to look up without the leading slash.
+            # Otherwise, we add the leading slash
+            if "/" in resource_name:
```

Comment:
Might be overkill, but

```suggestion
            if "/" in resource_name and not resource_name.startswith("/"):
```